README.md

```diff
@@ -0,0 +1,236 @@
+# Instructions
+
+To get the API up and running run:
+
+```console
+    yarn reload
+```
+
+All API requests are made to: ***http://localhost:8000***
+
+# MVP
+
+## Register
+
+a **POST** request to */api/staff/register* will create a new user and return an object including an authentication token:
+
+{
+    token: 'aKafj1Dg2kLD434So',
+    id: 1,
+    email: 'jdoe@abc.com',
+    message: 'Registration successful.'
+}
+
+the **POST** request must include the following:
+
+- *name*
+- *email*
+- *password*
+
+
+## Login
+
+a **POST** request to */api/staff/login* will login the user and return an object with an authentication token:
+
+{
+    token: 'aKafj1Dg2kLD434So',
+    id: 1,
+    email: 'jdoe@abc.com',
+    message: 'User logged in successfully.'
+}
+
+the **POST** request must include the following:
+
+- *email*
+- *password*
+
+
+## Items
+
```

Comment:
Really awesome job fleshing out this readme! For the purposes of displaying images in the cards, I'd like the object formatting to look more like:

```
let items = [
  {
    id: 0,
    name: 'apples',
    quantity: 100,
    unit: 'oz',
    imageUrl: "https://i.imgur.com/pMfBRRt.jpg",
  },
  {
    id: 1,
    name: 'bananas',
    quantity: 200,
    unit: 'oz',
    imageUrl: "https://i.imgur.com/9ny4c0E.jpg"
  },
  {
    id: 2,
    name: 'potatoes',
    quantity: 10,
    unit: 'lb(s)',
    imageUrl: "https://i.imgur.com/47fHnED.jpg"
  },
  {
    id: 3,
    name: 'carrots',
    quantity: 13,
    unit: 'lb(s)',
    imageUrl: "https://i.imgur.com/RTZ0qFP.jpg"
  },
  {
    id: 4,
    name: 'eggs',
    quantity: 31,
    unit: 'dozen',
    imageUrl: "https://i.imgur.com/bQYFZjw.jpg"
  },
  {
    id: 5,
    name: 'broccoli',
    quantity: 30,
    unit: 'oz',
    imageUrl: "https://i.imgur.com/dRF2mfc.jpg"
  },
];
```
And you can have categoryIDs in there too.